backend/src/routes/helpers/validation.ts

```diff
@@ -0,0 +1,9 @@
+import * as z from 'zod';
+
+export const stringToNumberTransformer = z.transformer(
```

Comment:
validates input > transforms > validates output of transform.

e.g. will throw error on input 'ABC'

Reply:
Nit: I generally prefer to export at the end of the module, I think it makes exports easier to track. If you have a good reason for exporting a const, then feel free to ignore.

Reply:
ok